src/transformers/models/whisper/feature_extraction_whisper.py

```diff
@@ -0,0 +1,262 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Feature extractor class for Whisper
+"""
+
+from typing import List, Optional, Union
+
+import numpy as np
+import torch
```

Comment:
Need to wrap this into a "if torch is available" dependency

Reply:
Please, let's not mark these comments as "resolved" if they are not

Reply:
I am pretty sure I wraped torch with if is_torch_available():